### ISSUE TITLE
Admin Category List - Long List Overlaps Debug Section

### ISSUE THREAD

**Issue by Dirty-Butter** [cubecart/v6#55]
See attached
![v6acp category list overlap 2015-01-12 15-33-37](https://cloud.githubusercontent.com/assets/6146505/5711514/c71d3086-9a70-11e4-92f1-511f816f679f.jpeg)

**Comment by Dirty-Butter** [comment]
PS - shows lack of separator for Category>Subcategory creating strange category names. This was from a test site using my current product data.

**Comment by bhsmither** [comment]
I see two solutions: make the appropriate div block grow taller, or start a scroller.

(Pagination?)

In /js/admin.js, line 292:
```
$('#page_content').css('min-height',nav_height+100);
/* $('#page_content').height(nav_height+100); */
```

**Comment by bhsmither** [comment]
Or in the admin template file main.php, line 71:
`<div id="page_content" style="overflow: auto;">`

**Comment by Dirty-Butter** [comment]
I don't think I want pagination. It's helpful to see the whole list at once. I would prefer div block growing, but can adapt to whatever seems best.

**Comment by bhsmither** [comment]
Or, overflow the table so as to keep the Save buttons visible (if your browser document window height is tall enough - I have several toolbars).

**Comment by Dirty-Butter** [comment]
I don't always understand the distinction of the various terms. What I had in mind was to be able to see the whole list, with the Save button at the bottom, simply by going up and down the page with the mouse, etc.

**Comment by bhsmither** [comment]
I understand. Aside from the pagination issue, the one page that has always irked me is the PHP Info page. After scrolling down to the bottom of that page, it was tedious (however minor) to get back to the top so I could go to a different admin page.

Ideally, the Nav Stack would be fixed in the browser window and everything else scrolls up and down.

**Comment by abrookbanks** [comment]
Fixed. Thanks for this. Category separator issue fixed with commit f86bafe4de4b6e6003bd246508c9a6d064214170
